### DISCUSSION TITLE
Microstructure (orientation) generation for elongated grains

### DISCUSSION THREAD

**Discussion by makeabhishek** [mtex-toolbox/mtex#1151] [Ask Anything]
Hi, 
Is it possible to generate a microstructure (Euler angles) with equiaxed/elongated grains for a random texture and preferred orientation in MTEX for a given statistics (eg. mean grain size), just like DREAM3D or NEPER software?

Regards
Abhishek

**Comment by kilir** [comment]
Hi,
could you explain how to desribe a microstructure (microstructure to my understanding: grain shapes, sizes, boundaries..) solely based on Euler angles?

Cheers,
Rüdiger

**Reply by makeabhishek** [reply]
Dear Rüdiger,
My apologies for the incomplete description. Yes, microstructure means grain shapes, sizes, and boundaries. First, generate the microstructure based on grain statistics and then obtain the Euler angle for each grain. I think this is usually done by Voronoi tesselation.

Regards
Abhishek

**Comment by zmichels** [comment]
Just in the spirit of being pedantic…
Is crystallographic orientation not an aspect of microstructure? For instance, if you observed two different materials with similar grain shapes and sizes, but they had different crystallographic preferred orientations… I think that means they have different microstructures, no?

Z

**Reply by makeabhishek** [reply]
Yes, that's correct. this is what I want to generate. similar grain shapes with random crystallographic orientation. 


Abhi
